rules/legacy_xcodeproj.bzl

```diff
@@ -595,8 +595,14 @@ def _header_search_paths_for_target(target_name, all_transitive_targets):
     header_search_paths.append("\"$BAZEL_WORKSPACE_ROOT\"")
     return " ".join(header_search_paths)
 
-def _swift_copts_for_target(target_name, all_transitive_targets):
+def _swift_copts_for_target(target_name, all_transitive_targets, mixed = False):
     copts = []
+    if mixed:
```

Comment:
The fact that this never read `copts` is one of the larger issues with it but will go away when we remove this code. This seems fine for now